src/Robo/Commands/Tests/RunTestsCommand.php

```diff
@@ -115,6 +115,10 @@ public function executeTests() {
           $task->dieOnFail();
         }
 
+        if (isset($test['directory'])) {
+          $task->directory($test['directory']);
```

Comment:
nit: the options in this file are arranged by `@param` type — maybe move this to where he other strings live.